src/app/shared/domain/album.domain.ts

```diff
@@ -20,6 +20,7 @@ export interface Albums {
   year: string;
   genre: string;
   playCount: number;
+  coverArt: number;
```

Comment:
`coverArt` is a `string` in the [schema](http://www.subsonic.org/pages/inc/api/schema/subsonic-rest-api-1.16.1.xsd).